File: erpnext/crm/report/call_log/call_log.py
# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
# For license information, please see license.txt
from __future__ import unicode_literals
import frappe
from frappe import _

def execute(filters=None):
	columns, data = [], []
	columns=get_call_log_columns()
	# data=get_call_log_data()
	return columns, data
def get_call_log_columns():
	return [
		_("Campaign Name") + ":data:130", 
		_("Lead Count") + ":Int:80",
		_("Opp Count") + ":Int:80",
		_("Quot Count") + ":Int:80", 
		_("Order Count") + ":Int:100",
		_("Order Value") + ":Float:100",
		_("Opp/Lead %") + ":Float:100",
		_("Quot/Lead %") + ":Float:100",
		_("Order/Quot %") + ":Float:100"
	]
def get_call_log_data():
	return [
		{"Staff Name":"test", 
		"No. of call" :3, 
		"No. of incoming calls" :2, 
		"No. of outgoing call":1}
	]

File: erpnext/crm/report/call_log/call_log.js
// Copyright (c) 2016, Frappe Technologies Pvt. Ltd. and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["Call Log"] = {
	"filters": [
			{
				"fieldname": "month",
				"label": "Month",
				"fieldtype": "Select",
				"options": ["January","February","March","April","May","June","July","August","September","October","November","December"],
			},
			{
				"fieldname": "year",
				"label": "Year",
				"fieldtype": "Select",
				"options": ["2017","2018","2019","2020","2021","2022","2023","2024","2025"],
			}
	]};

